### PR TITLE
R3.5

### PR DESCRIPTION
**Pull request by cbrueffer**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,5 +1,4 @@
-{% set version = "1.1.2" %}
-
+{% set version = '1.1.2.1' %}
```
Comment (ocefpaf) [inline recipe/meta.yaml:1]: Sorry. I missed that you upgraded it here too and merged #3  :-/
Can you rebase this?

Comment (cbrueffer) [reply recipe/meta.yaml:1]: Wow, bad timing! :-D I'll rebase now.

Comment (ocefpaf) [reply recipe/meta.yaml:1]: Yeah. I saw your PR and wanted to make it so we would get the latest version too. Sorry :grimacing:

Comment (cbrueffer) [reply recipe/meta.yaml:1]: No worries! I always update to the latest version as part of these rebuilds FYI.

**Comment by bgruening** [comment]
Top!

**Pull request merged**